.pre-commit-config.yaml

```diff
@@ -70,54 +70,138 @@ repos:
-                aiida/orm/implementation/comments.py|
-                aiida/orm/implementation/computers.py|
-                aiida/orm/implementation/groups.py|
-                aiida/orm/implementation/logs.py|
-                aiida/orm/implementation/nodes.py|
-                aiida/orm/implementation/users.py|
-                aiida/orm/implementation/querybuilder.py|
-                aiida/orm/entities.py|
-                aiida/orm/extras.py|
-                aiida/orm/authinfos.py|
+                .github/.*|
+                .molecule/.*|
+                docs/.*|
+                utils/.*|
+
+                aiida/calculations/arithmetic/add.py|
+                aiida/calculations/diff_tutorial/calculations.py|
+                aiida/calculations/templatereplacer.py|
+                aiida/calculations/transfer.py|
+                aiida/cmdline/commands/cmd_archive.py|
+                aiida/cmdline/commands/cmd_calcjob.py|
+                aiida/cmdline/commands/cmd_code.py|
+                aiida/cmdline/commands/cmd_computer.py|
+                aiida/cmdline/commands/cmd_data/cmd_list.py|
+                aiida/cmdline/commands/cmd_data/cmd_upf.py|
+                aiida/cmdline/commands/cmd_devel.py|
+                aiida/cmdline/commands/cmd_group.py|
+                aiida/cmdline/commands/cmd_node.py|
+                aiida/cmdline/commands/cmd_run.py|
+                aiida/cmdline/commands/cmd_shell.py|
+                aiida/cmdline/commands/cmd_storage.py|
+                aiida/cmdline/groups/dynamic.py|
+                aiida/cmdline/params/options/commands/setup.py|
+                aiida/cmdline/params/options/interactive.py|
+                aiida/cmdline/params/options/main.py|
+                aiida/cmdline/params/options/multivalue.py|
+                aiida/cmdline/params/types/group.py|
+                aiida/cmdline/params/types/plugin.py|
+                aiida/cmdline/utils/ascii_vis.py|
+                aiida/cmdline/utils/common.py|
+                aiida/cmdline/utils/echo.py|
+                aiida/common/extendeddicts.py|
+                aiida/common/hashing.py|
+                aiida/common/utils.py|
+                aiida/engine/daemon/execmanager.py|
+                aiida/engine/processes/calcjobs/manager.py|
+                aiida/engine/processes/calcjobs/monitors.py|
+                aiida/engine/processes/calcjobs/tasks.py|
+                aiida/engine/processes/control.py|
+                aiida/engine/processes/functions.py|
+                aiida/engine/processes/ports.py|
+                aiida/manage/configuration/__init__.py|
+                aiida/manage/configuration/config.py|
+                aiida/manage/configuration/profile.py|
+                aiida/manage/configuration/settings.py|
+                aiida/manage/external/rmq/launcher.py|
+                aiida/manage/tests/main.py|
+                aiida/manage/tests/pytest_fixtures.py|
```

Comment:
why are the lines
```
                aiida/orm/comments.py|
                aiida/orm/computers.py|
```
not removed? If they were properly typed before, should not exclude them now, should we?
Same for
```
                aiida/tools/graph/graph_traversers.py|
                aiida/tools/groups/paths.py|
```
further down

Reply:
see https://github.com/aiidateam/aiida-core/pull/5811/#issuecomment-1338208930